lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -469,6 +469,38 @@ CRI_HMRC_KBV_J6:
     next:
       targetState: PYI_NO_MATCH
 
+# No photo id M2B journey (J7)
+CRI_BANK_ACCOUNT_J7:
+  response:
+    type: cri
+    criId: bav
+    context: bank_account
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: CRI_NINO_WITH_SCOPE_J7
+
+CRI_NINO_WITH_SCOPE_J7:
+  response:
+    type: cri
+    criId: nino
+    scope: identityCheck
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: ADDRESS_AND_FRAUD_J7
+
+ADDRESS_AND_FRAUD_J7:
+  nestedJourney: ADDRESS_AND_FRAUD
+  exitEvents:
+    next:
+      targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
+      checkFeatureFlag:
```

Comment:
Hmm shouldn't this be `checkIfDisabled` so we're checking the enabled flag of the HMRC KBV CRI? (and flip the states around)

Reply:
I took this from Joe's talk on Feature Flags yesterday and it seems to work when I've tested it. We seem to be using both methods in this repo, is there a preference for one over the other?

Reply:
I think the feature flags and enabled/disabled CRIs are subtly different:
- The `checkIfDisabled` checks should be there long-term, so we can disable CRIs and the journey map updates accordingly
- The `checkFeatureFlag` checks are (typically) shorter-lived, so we can introduce new functionality without risking it going live

IMO, if it's part of the requirements that we fall back to experian KBVs if HMRC is disabled, then we should be doing that with `checkIfDisabled`.

We may additionally want to `checkFeatureFlag` as well (noting that you can nest these conditions!), but it might not be necessary if they'll both be turned on simultaneously.